server/chat-plugins/github.ts

```diff
@@ -0,0 +1,266 @@
+/**
+ * Chat plugin to view GitHub events in a chatroom.
+ * By Mia, with design / html from xfix's original bot, https://github.com/xfix/GitHub-Bot-Legacy/
+ * @author mia-pi-git
+ */
+
+import {FS, Utils} from '../../lib';
+
+const STAFF_REPOS = ['pokemon-showdown', 'pokemon-showdown-client', 'Pokemon-Showdown-Dex'];
```

Comment:
This feels very Main-specific. Can we put it in a config file or let Upper Staff configure it with a command?

Reply:
I don't think it needs to be US, but yeah a command would probably be better.

Reply:
Set it to a Config setting.